src/mmw/apps/geoprocessing_api/tasks.py

```diff
@@ -47,9 +48,21 @@ def start_rwd_job(location, snapping, data_source):
     rwd_url = 'http://%s:%s/%s/%f/%f' % (RWD_HOST, RWD_PORT, end_point,
                                          lat, lng)
 
+    params = {}
+
     # The Webserver defaults to enable snapping, uses 1 (true) 0 (false)
     if not snapping:
-        rwd_url += '?snapping=0'
+        params['snapping'] = 0
+
+    # RWD also defaults to simplify the shape according to a tolerance.
+    # Passing it `?simplify=0` returns the unsimplified result.
+    if not (simplify is False):
```

Comment:
For easier reading, I think this can be simply:
`if simplify is not False`

Reply:
Pushed c8242db to make this change.